book/templating.rst

```diff
@@ -867,6 +867,20 @@ configuration:
 
 .. configuration-block::
 
+    .. code-block:: php-annotations
+
+        // src/AppBundle/Controller/WelcomeController.php
+        class WelcomeController extends Controller
```

Comment:
At the beginning of the annotations snippets we should put the `use` imports needed to make the code work:

```php
use Symfony\Bundle\FrameworkBundle\Controller\Controller;
use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;

// ...
```